samples/embedded-auth-with-sdk/src/main/java/com/okta/spring/example/controllers/LoginController.java

```diff
@@ -219,6 +224,38 @@ public ModelAndView selectAuthenticator(final @RequestParam("authenticator-type"
         }
     }
 
+    /**
+     * Send poll request for enrollment authenticators.
+     *
+     * @param session the session
+     * @return select authenticator view or select factor view or null
+     */
+    @GetMapping("/poll")
+    @ResponseBody
+    private RemediationOption sendPollRequest(HttpSession session) {
```

Comment:
We don't want the client to deal with `RemediationOption` object. We should probably stick the polling information inside `AuthenticationResponse` and let the client access it from there instead of directly exposing `RemediationOption` object.